artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPMessage.java

```diff
@@ -971,7 +971,7 @@ public Object getAMQPUserID() {
 
    @Override
    public Object getDuplicateProperty() {
-      return null;
+      return getObjectProperty(org.apache.activemq.artemis.api.core.Message.HDR_DUPLICATE_DETECTION_ID);
```

Comment:
Surely this should not be Artemis specific but some amqp value

Reply:
I wasn't aware that there was an AMQP-specific duplicate detection property/annotation. Do you have documentation on that?

Reply:
I dont know if its amqp specific. I just know they do it by message id, which is a more agnostic to vendor field.

Reply:
Nothing here precludes Service Bus from continuing to de-dupe by message ID is there?

Reply:
nope. It was more a query if there was any form of standard, i assumed maybe there was based on azure having same feature. But if there isn't and its vendor specific there. Then for now we just have to be vendor specific to our keys too. +1 this PR